### PR TITLE
Fix some of the French copy

### PR DESCRIPTION
**Pull request by abusque**
Correct some typos, small mistakes, and phrasing issues in the French
copy.

Signed-off-by: Antoine Busque <antoinebusque@gmail.com>

### CODE REVIEW

File: webapp/content.py [hunk 1]
```diff
@@ -120,12 +120,12 @@ def what_is_quebec_drinking(self):
     def recommendations(self):
         return '''
             Nous avons analysé l’inventaire en ligne de la SAQ, histoire de comprendre comment la 
-            population suit les recommandations de Legault.
+            population suit les recommandations du Premier Ministre Legault.
```
Comment (abusque) [inline webapp/content.py:123]: This is just to take the informal-ness down a notch, more in line with the rest of the copy.

File: webapp/content.py [hunk 2]
```diff
@@ -164,24 +164,24 @@ def internationally(self):
...
         return '''
-                Restez à la maison, ouvrez une bonne bouteille, et dites vous que tout le Québec lève son verre ensemble! 
+                Restez à la maison, ouvrez une bonne bouteille, et dites-vous que tout le Québec lève son verre ensemble!
                 '''
 
     @property
     def footer(self):
         return Markup("""
-                <strong>Un Verre de Vin Peut Aider </strong> est fait par
+                <strong>Un Verre de Vin Peut Aider </strong> a été conçu et réalisé par
```
Comment (abusque) [inline webapp/content.py:173]: Original wasn't technically incorrect here, but I think this phrasing is more idiomatic.

File: webapp/content.py [hunk 3]
```diff
@@ -164,24 +164,24 @@ def internationally(self):
...
                 <a href="https://www.linkedin.com/in/archy-de-berker/"> 
                 Archy de Berker </a> et <a href="https://www.linkedin.com/in/claudelrheault/">
                  Claudel Rheault</a> avec l'aide de
                 <a href="https://www.linkedin.com/in/steph-willis-7465759a/"> Stephanie Willis</a>
                 <p class='is-size-6 is-family-serif'>
                 Nous aimerions être en contact;  
                 <a href="https://ctt.ac/68jVg">
-                 tweet-nous un photo </a> avec un grand verre de ta vintage preferée! 
+                 Envoyez-nous une photo  sur Twitter</a> avec un grand verre de votre vin preferé!
```
Comment (abusque) [inline webapp/content.py:181]: In this context, "vintage" would be translated as "millésime", but it doesn't really have the same meaning as the English version (too specific, really only means a specific year of a specific wine), so I went with the more generic version here.

Also "tweet" as a verb in French is a bit awkward hence the rewording there.

**Comment by archydeberker** [comment]
Tremendous, thanks @abusque

**Pull request merged**